pr-deploy/src/app.ts

```diff
@@ -79,20 +82,20 @@ function initializeDeployment(app: Application) {
     }
 
     const pr = new PullRequest(
-      context.github,
-      context.payload.check_run.head_sha,
+      (context.github as unknown) as Octokit,
+      context.payload.check_run.head_sha
     );
     await pr.deploymentInProgress();
     const travisBuild = await pr.getTravisBuildNumber();
     await unzipAndMove(travisBuild)
-      .then(bucketUrl => {
-        pr.deploymentCompleted(
+      .then(async bucketUrl => {
+        await pr.deploymentCompleted(
           bucketUrl,
           `${BASE_URL}amp_dist_${travisBuild}/`
         );
       })
-      .catch(e => {
-        pr.deploymentErrored(e);
+      .catch(async e => {
+        await pr.deploymentErrored(e);
```

Comment:
Nit: Since you're already tweaking this, consider making it all async

```javascript
try {
  const bucketUrl = await unzipAndMove(travisBuild);
  await pr.deploymentCompleted(bucketUrl, `${BASE_URL}amp_dist_${travisBuild}/`);
} catch (e) {
  await pr.deploymentErrored(e);
}
```